### PR TITLE
feat(): create about page setup

### PR DESCRIPTION
**Pull request by im-klau**
## Name PR according scheme: type of changes (feat/fix/refactor etc.) + description
feat(): create about page setup

## Jira issue

JIRA issue: [FSU-156](https://klau.atlassian.net/browse/FSU-156)

## Describe your changes
This PR introduces basic Next.js setup for a new "About" page.

Verify that the page exists: [About page](https://sterczaceuszy-git-about-page-setup-fsu.vercel.app/fundacja) 👀 

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/templates/About/About.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <Styled.Main>
+      <Container>
+        <Typography variant="h4" as="h3">
+          ♥️ {supporting.total} Pomocnych serc
+        </Typography>
+        <Typography variant="h4" as="h3">
+          🏡 {adoptedAnimals.total} Znalezionych domów
+        </Typography>
+        <Typography variant="h4" as="h3">
+          🏡 {animals.total + adoptedAnimals.total} Uratowanych zwierząt
```
Comment (AmandaKr) [inline packages/frontend/app/templates/About/About.jsx:46]: Here a animal icon would fit better I think :D

**Review by AmandaKr — APPROVED** [review]
🚀

File: packages/frontend/app/templates/About/About.jsx (hunk 1 above)
Comment (im-klau) [reply packages/frontend/app/templates/About/About.jsx:46]: Probably, bu it's a temporary solution :D

**Review by nata-kab — COMMENTED** [review]
🚀

**Review by nata-kab — APPROVED** [review]
🚀

**Pull request merged**